packages/aragon-api-react/src/index.js

```diff
@@ -51,13 +57,23 @@ function AragonApi({
 
     let subscribers = []
 
-    const handleMessage = ({ data }) => {
-      if (data.from !== 'wrapper') {
+    const handleMessage = ({ data } = {}) => {
+      if (!data || data.from !== 'wrapper') {
         return
       }
+
       if (data.name === 'displayMenuButton') {
         setDisplayMenuButton(data.value)
       }
+
+      if (data.name === 'path' && data.value !== path) {
+        setPath(data.value)
+      }
+
+      if (data.name === 'apps' && Array.isArray(data.value)) {
```

Comment:
Is this message already being sent?

Reply:
This part has been replaced by `api.getApps()`.